susecloud

```diff
@@ -285,6 +289,16 @@ else
 fi
 
 #############################################################
+section_header "Ceph report"
+
+if [ -e /usr/bin/ceph ]; then
+    now=$(date +%s)
+    ceph_report_name=$TMPLOGS_DIR/ceph_report_${now}.txt
+    ceph report > $ceph_report_name
```

Comment:
Redirecting stderr would be also nice, look on https://github.com/SUSE-Cloud/supportutils-plugin-suse-openstack-cloud/pull/47/files